src/interfaces.ts

```diff
@@ -19,12 +19,12 @@ export interface UserSignupPosting {
 }
 
 export interface Action {
-  id?: number,
+  id: number,
   name: string
 }
 
 export interface Brainstorm {
-  id?: number,
+  id: number,
   question: string,
   idea: string,
   action: string,
```

Comment:
I have added some interfaces to this file as well. May have some overlap, but I think it makes sense to have distinct interfaces for the Brainstorm Form vs the finished Brainstorm